src/main/java/com/bean00/MessageController.java

```diff
@@ -1,41 +1,78 @@
 package com.bean00;
 
-import java.io.BufferedReader;
 import java.io.IOException;
-import java.io.PrintWriter;
+import java.io.Writer;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 
 public class MessageController {
```

Comment:
I think this class is starting to take on more responsibilities. I'd like to see this be improved in future PRs!

Reply:
Ok, sounds good